app/build.gradle

```diff
@@ -60,4 +60,10 @@ dependencies {
     androidTestImplementation 'com.android.support.test:rules:1.0.2'
     androidTestImplementation 'com.android.support.test:runner:1.0.2'
     androidTestImplementation 'androidx.test.espresso:espresso-contrib:3.1.1'
+
+    // Visual Studio App Center
+    def appCenterSdkVersion = '2.0.0'
```

Comment:
Does Java constant convention not apply here?

Also is there a way to make this a constant?

Reply:
Oh wait is def the version of constant?

Reply:
So like APP_CENTER_SDK_VERSION

Reply:
Gradle convention is to write it in camelCase, the code you're looking at is from Microsoft documentation